arm_wiki/Alternate-Install-TrueNAS.md

```diff
@@ -0,0 +1,69 @@
+# IMPORTANT
+
+***
+This installation method is not supported or maintained by the ARM Developers.
+For full support and continued maintenance,
+recommend installing ARM via the supported [Docker Container](https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/docker).
```

Comment:
I'm still confused about this, because we are installing the provided Docker Container... just not via the install script (which wouldn't work on TrueNAS anyway).